apps/server/src/connect/dataEndpoints.test.ts

```diff
@@ -397,6 +402,77 @@ describe("dataEndpoints", () => {
         expect(res.send).toHaveBeenCalledWith("Something went wrong");
         expect(res.status).toHaveBeenCalledWith(400);
       });
+
+      it("returns transaction data, if it's TestAdapterB and passes the transaction validator", async () => {
```

Comment:
These it statements are hard to parse. When looking at the code it should be easy for me to look through the tests and see if there's coverage for everything. We don't need to mention which aggregator it's for except for sophtron. the it statements should focus on the code that's being tested instead of the aggregator that's being tested. 

Examples: "succeeds if there is a custom validator and it passes" "succeeds if there's no custom validator" "fails if a custom validator fails" "fails if it's sophtron and it's missing a startTime" "succeeds if it's sophtron and it has a startTime" etc